src/lib/terkin/driver/hx711_sensor.py

```diff
@@ -86,10 +89,15 @@ def start(self):
         gain = self.parameter.get('gain', 128)
         scale = self.parameter['scale']
         offset = self.parameter['offset']
+        scaleB = self.parameter['scaleB']
+        offsetB = self.parameter['offsetB']
 
         # Initialize the HX711 hardware driver.
         log.info('Initializing HX711 sensor with '
                  'pin_dout={}, pin_pdsck={}, gain={}, scale={}, offset={}'.format(pin_dout, pin_pdsck, gain, scale, offset))
+        if self.parameter['dualchannel']:
```

Comment:
Please say `self.parameter.get('dualchannel')` here to improve backwards-compatibility.

Reply:
Please explain: backward compatible to what? The parameter is created in includeme() and the default is false so it works even if it is not set in settings.py.

Reply:
So true. Nevermind then and sorry for the noise!